### DISCUSSION TITLE
Error saat melakukan print(llm.predict(prompt))

### DISCUSSION THREAD

**Discussion by widyaarahma** [teamalgoritma/community#881] [Q&A] [answered]
InvalidRequestError: The model `text-davinci-003` has been deprecated, learn more here: https://platform.openai.com/docs/deprecations

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear @widyaarahma 

Berdasarkan pesan error, sepertinya model yang ingin Anda gunakan sudah deprecated. Merujuk pada link yang dilampirkan, Anda dapat mencoba model `gpt-3.5-turbo-instruct`

Semoga jawaban ini dapat membantu,

Best regards,
Team Algoritma

**Reply by widyaarahma** [reply]
yang perlu diganti bagian mana ya kak?

**Reply by saskia-dwi-ulfah** [reply]
Pada saat ibu mendefinisikan `llm`-nya bu

**Reply by ramdhinafinita** [reply]
Halo @widyaarahma , untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.
